File: Retos/Reto #40 - TABLA DE MULTIPLICAR [Fácil]/typescript/JherysVargas.ts
const getNumber = (): number => {
  const value = prompt("Ingrese un número");

  return Number(value);
};

const generateMultiplicationTable = (): void => {
  const value: number = getNumber();

  for (let i = 1; i <= 10; i++) {
    console.log(`${value} x ${i} = ${value * i}`);
  }
};

generateMultiplicationTable();

File: Retos/Reto #40 - TABLA DE MULTIPLICAR [Fácil]/dart/JherysVargas.dart
import 'dart:io';

void main(List<String> arguments) {
  generateMultiplicationTable();
}

int getNumber() {
  print('Ingrese un número:');
  return int.parse(stdin.readLineSync()!);
}

void generateMultiplicationTable() {
  final int value = getNumber();

  for (var i = 1; i <= 10; i++) {
    print('$value x $i = ${value * i}');
  }
}
